lib/streamlit/delta_generator.py

```diff
@@ -77,14 +79,21 @@
 
 # List of Streamlit commands that perform a Pandas "melt" operation on
 # input dataframes.
-DELTAS_TYPES_THAT_MELT_DATAFRAMES = ("line_chart", "area_chart", "bar_chart")
+DELTA_TYPES_THAT_MELT_DATAFRAMES = ("line_chart", "area_chart", "bar_chart")
+BETA_DELTA_TYPES_THAT_MELT_DATAFRAMES = (
+    "beta_line_chart",
+    "beta_area_chart",
+    "beta_bar_chart",
+)
```

Comment:
These need a unit test (looks like the original unit test is in `add_rows_test.py: test_deltas_that_melt_dataframes()`, and can probably be mirrored).

Reply:
Done